src/main/java/org/jabref/model/database/BibDatabase.java

```diff
@@ -613,4 +613,15 @@ public String generateSharedDatabaseID() {
     public DuplicationChecker getDuplicationChecker() {
         return duplicationChecker;
     }
+
+    public List<BibEntry> getEntriesInGroup(String groupName) {
```

Comment:
The GroupsTreeNode class already contains the `getMatchingGroups` method https://github.com/JabRef/jabref/blob/master/src/main/java/org/jabref/model/groups/GroupTreeNode.java#L164, which slightly differs from this code here but is very similar in its purpose (its also matches dynamics and search groups). Thus I would propose to move the new code to a new method `GroupTreeNode::getMatchingExplicitGroups`. Moreover, please reuse `ExplicitGroup:contains` https://github.com/JabRef/jabref/blob/master/src/main/java/org/jabref/model/groups/WordKeywordGroup.java#L105 for the group membership check.

Reply:
Ok, thanks. I'll look into these things. I added this method basically because I'm not at all an expert in the group code and didn't know about these methods.

Reply:
Moved and refactored. I kept the method name, though.